SU-SOE/stanford_bean_types
# 7.x 2.x postcard linked bean

# READY FOR REVIEW

# Summary
- This is the new postcard-linked bean type. Similar to the postcard bean type. Using the default display mode, the title and the image are links.
This was separated from the postcard bean type due to technical issues getting the Field Group Link module to work with the Display Suite module. 😞 

# Needed By (Date)
- Last week.

# Urgency
- Yup

# Steps to Test

- Make sure you have Image style: Header (scale and crop 370 x 170) 
- Pull this branch
- Flush the cache
- Revert this feature
- Create a "Stanford Postcard Linked" block
- Place block
- Verify html looks right

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1979

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
